prow/plugins/trigger/trigger.go

```diff
@@ -31,9 +35,29 @@ const (
 )
 
 func init() {
-	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment, nil)
-	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, nil)
-	plugins.RegisterPushEventHandler(pluginName, handlePush, nil)
+	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment, helpProvider)
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, helpProvider)
+	plugins.RegisterPushEventHandler(pluginName, handlePush, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	configInfo := map[string]string{}
+	for _, repo := range enabledRepos {
+		parts := strings.Split(repo, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
+		}
+		trusted := trustedOrgForRepo(config, parts[0], parts[1])
+		configInfo[repo] = fmt.Sprintf("The trusted github organization for this repository is %q.", trusted)
+	}
+	return &pluginhelp.PluginHelp{
+			Description: "The trigger plugin creates ProwJobs in reaction to commands and pull request events. It is responsible for ensuring that jobs are only run on trusted PRs. A PR is considered trusted if the author is a member of the 'trusted organization' for the repo or if such a member has left an `/ok-to-test` command on the PR. Trigger creates jobs automatically when a new trusted PR is created or when an untrusted PR becomes trusted, but it can also be used to create jobs manually via the `/test` command. The `/retest` command can be used to rerun jobs that have reported failure.",
```

Comment:
Users of these commands will not necessarily be familiar with the concept of a ProwJob. I suggest we reword "creates ProwJobs" to "starts tests".

Reply:
Done.